minio/api.py

```diff
@@ -181,7 +180,6 @@ def __init__(self, endpoint, access_key=None,
                     Static(access_key, secret_key, session_token),
                     EnvAWS(),
                     EnvMinio(),
-                    IamEc2MetaData(),
```

Comment:
Well, this is a breaking change, we should not forget about mentioning it when we release minio-py

Reply:
Yes. I think we are covering in https://github.com/minio/minio-py/issues/902